zdict/tests/dictionaries/test_oxford.py

```diff
@@ -0,0 +1,360 @@
+from pytest import raises
+from unittest.mock import Mock, mock_open, patch
+
+from zdict.dictionaries.oxford import OxfordDictionary
+from zdict.exceptions import NotFoundError, QueryError
+from zdict.models import Record
+from zdict.zdict import get_args
+
+
+class TestOxfordDictionary:
+    def setup_method(self, method):
+        self.dict = OxfordDictionary(get_args())
+
+    def teardown_method(self, method):
+        del self.dict
+
+    def test_provider(self):
+        assert self.dict.provider == 'oxford'
+
+    def test__get_url(self):
+        uri = 'https://od-api.oxforddictionaries.com/api/v1/entries/en/mock'
+        assert self.dict._get_url('mock') == uri
+
+    @patch('os.path.exists')
+    def test__get_app_key(self, exists):
+        key_file = mock_open(read_data="""
+            test_app_id,
+            test_app_key """)
+
+        with patch('builtins.open', key_file):
+            app_id, app_key = self.dict._get_app_key()
+
+        assert app_id == 'test_app_id'
+        assert app_key == 'test_app_key'
+
+    def test__show_instruction(self):
```

Comment:
I think we have to make sure `self._show_instruction()` will (only?) be executed when `KEYFILE` doesn't exist.